src/main/java/uk/gov/ons/ssdc/supporttool/rasrm/service/RasRmSampleSetupService.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.ons.ssdc.supporttool.rasrm.service;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.ssdc.common.model.entity.CollectionExercise;
+import uk.gov.ons.ssdc.supporttool.model.repository.CollectionExerciseRepository;
+import uk.gov.ons.ssdc.supporttool.rasrm.client.RasRmSampleServiceClient;
+
+@Component
+public class RasRmSampleSetupService {
+  private static final Set<String> MANDATORY_RAS_RM_BUSINESS_COLLEX_METADATA =
+      Set.of("rasRmCollectionExerciseId", "rasRmCollectionInstrumentId");
+
+  private final RasRmSampleServiceClient rasRmSampleServiceClient;
+  private final CollectionExerciseRepository collectionExerciseRepository;
+
+  public RasRmSampleSetupService(
+      RasRmSampleServiceClient rasRmSampleServiceClient,
+      CollectionExerciseRepository collectionExerciseRepository) {
+    this.rasRmSampleServiceClient = rasRmSampleServiceClient;
+    this.collectionExerciseRepository = collectionExerciseRepository;
+  }
+
+  public void setupSampleSummary(CollectionExercise collectionExercise, int sampleCaseCount) {
```

Comment:
Not sure about the wording, is this 'setUpSampleSummary' also posting it to RasRM as well?

Reply:
I thought that because it's in the `RasRmSampleSetupService` class, it was clear that this is setting stuff up in RAS-RM